app/services/doc_auth/lexis_nexis/responses/lexis_nexis_response.rb

```diff
@@ -0,0 +1,103 @@
+module DocAuth
+  module LexisNexis
+    module Responses
+      class LexisNexisResponse < DocAuth::Response
+        PII_DETAILS = %w[
+          Age
+          DocIssuerCode
+          DocIssuerName
+          DocumentName
+          DOB_Day
+          DOB_Month
+          DOB_Year
+          FullName
+          Sex
+          Fields_Address
+          Fields_AddressLine1
+          Fields_AddressLine2
+          Fields_City
+          Fields_State
+          Fields_PostalCode
+          Fields_Height
+        ].map(&:freeze).freeze
```

Comment:
I was wondering why map & freeze the strings on `PII_DETAILS` when `# frozen_string_literal: true` would do this for the code with ruby 2.3 (IIRC) and beyond. Do you have any constraint to use the `frozen_string_literal` magic comment?

Reply:
good call - thanks!